### DISCUSSION TITLE
Possible for Map Modifiers?

### DISCUSSION THREAD

**Discussion by Slipperyjim8** [xanthics/poe_gen_gwennen#4] [Ideas]
Is something like this possible for map modifiers?
Are the regex snippets automatically generated or are you doing them manually?

**Comment by xanthics** [comment]
This can be done with anything, as long as there is a good data source. https://github.com/xanthics/poe_gen_gwennen/blob/master/generate_ngrams.py pregenerated candidate ngrams for the web page.

**Comment by xanthics** [comment]
Can give it a try here: https://xanthics.github.io/poe_gen_search_string/
